py/torch_tensorrt/dynamo/types.py

```diff
@@ -0,0 +1,24 @@
+from typing import Sequence, Tuple
+
+# @manual=//deeplearning/trt/python:py_tensorrt
+import tensorrt as trt
+
+if hasattr(trt, "__version__"):
+    TRTNetwork = trt.INetworkDefinition
+    TRTTensor = trt.tensorrt.ITensor
+    TRTLayer = trt.ILayer
+    TRTPluginFieldCollection = trt.PluginFieldCollection
+    TRTPlugin = trt.IPluginV2
+    TRTDataType = trt.DataType
+    TRTElementWiseOp = trt.ElementWiseOperation
+else:
+    TRTNetwork = "trt.INetworkDefinition"
+    TRTTensor = "trt.tensorrt.ITensor"
+    TRTLayer = "trt.ILayer"
+    TRTPluginFieldCollection = "trt.PluginFieldCollection"
+    TRTPlugin = "trt.IPluginV2"
+    TRTDataType = "trt.DataType"
+    TRTElementWiseOp = "trt.ElementWiseOperation"
+
+Shape = Sequence[int]
+ShapeRange = Tuple[Shape, Shape, Shape]
```

Comment:
If these are being used by dynamo converters, keep it or else we can remove them.

Reply:
Grepped and checked it's not used. Removed in new commit